app/Providers/AppServiceProvider.php

```diff
@@ -30,5 +32,19 @@ public function boot()
         /** @var MailSettings $settings */
         $settings = $this->app->make(MailSettings::class);
         $settings->setConfig();
+
+
+        //get the current Branch name and latest git Version the Instance is running on and safe it as a config variable
+        try {
+            (Storage::get('latestVersion') ? config(["LATESTVERSION" => Storage::get('latestVersion')]) : config(["LATESTVERSION" => "0.0"]));
```

Comment:
only check the file and compare the file with the config value, don't store the lastest version in the config, this wil cause the config file to update which may result in ppl being unable to pull the updates using git until they merge their changes